server/sonar-server/src/test/java/org/sonar/server/project/ws/CreateActionTest.java

```diff
@@ -71,24 +73,26 @@
   public DbTester db = DbTester.create(system2);
   @Rule
   public UserSessionRule userSession = UserSessionRule.standalone();
+
   private DefaultOrganizationProvider defaultOrganizationProvider = TestDefaultOrganizationProvider.from(db);
   private ComponentUpdater componentUpdater = mock(ComponentUpdater.class, Mockito.RETURNS_MOCKS);
 
   private WsActionTester ws = new WsActionTester(
-      new CreateAction(
-          db.getDbClient(), userSession,
-          componentUpdater,
-          defaultOrganizationProvider));
+    new CreateAction(
+      new ProjectsWsSupport(db.getDbClient()),
+      db.getDbClient(), userSession,
+      componentUpdater,
+      defaultOrganizationProvider));
 
   @Test
   public void create_project() throws Exception {
```

Comment:
You can also test that project cannot be created in an organization if user has provisioning permission in another organization.